README.md

```diff
@@ -1,70 +1,54 @@
-# Getting Started with Create React App
+# Magic Books Inc
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This application is built with real-world business specifications. It an app for a Germany fictional company *Magic Books Inc*, the application helps organize and manage the company's bookstore. It is a fast, dynamic app built with ReactJs and a few more technologies listed below.
 
-## Available Scripts
+## Technologies used:
+> technologies used in building this project include but are not limited to: 
+- EcmaScript 6
+- webpack
+- big.js
+- reactJS
+- buildpack
 
-In the project directory, you can run:
+## live Demo
+[MagicBooks.Inc](https://magicbooks-inc.herokuapp.com/)
 
-### `yarn start`
+## Project Design
+![BookStore](./demo/projectdemo.gif)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Project SetUp
+You can clone the project on your local machine by running:
+- `git clone https://github.com/ClaytonSiby/Magic-Books-Inc.git`
+then:
+-  `cd The_Calculator`
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+### install all dependencies by running:
+- `npm install`
```

Comment:
## Changes instruction
- [ ] I believe you meant to have yarn here instead of npm for package management. 
```suggestion
- `yarn install`
```